docs/changelog/89775.yaml

```
@@ -0,0 +1,6 @@
+pr: 89775
+summary: Return 408 when open/start APIs time out
+area: Machine Learning
+type: enhancement
```

I think the previous return value of 500 in this situation should be considered a bug.  So I would change the release note to this:

```suggestion
summary: Return 408 instead of 500 when open/start APIs time out
area: Machine Learning
type: bug
```